### PR TITLE
[14.0][MIG] event_phone

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: event_phone/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module validate phone numbers in the Attendee form of the Event module, just like the
```
Comment (rvalyi) [inline event_phone/readme/DESCRIPTION.rst]: s /This module validate/This module validates/

Comment (alexis-via) [reply event_phone/readme/DESCRIPTION.rst]: fixed

Comment (rvalyi) [reply event_phone/readme/DESCRIPTION.rst]: not too sure what is the problem with Travis https://travis-ci.com/github/OCA/connector-telephony/jobs/505673501#L549 but it's not passing sadly..;

**Review by MeetKD — APPROVED** [review]
Code Review 👍

**Pull request merged**